readme.md

```diff
@@ -188,15 +182,50 @@ mbo.finalize().then(function(mediaBlobs) {
 });
 ```
 
+### Error Handling in finalize
+When *[finalize()](#finalize-method)* is called, the User Agent will perform these basic checks for the operations that are batched. This error checking should be done before executing any of the operations.
+
+For *[trim()](#trim-method)*
+1. Let *O* represent the blob to be trimmed
+2. If *startTime* is less than 0 **OR** *endTime* is greater than the *O*.duration **OR** *startTime* is greater than the *endTime*:
+    * Reject promise with a new [DOMException](https://heycam.github.io/webidl/#idl-DOMException)
```

Comment:
For all the new uses of DOMException, should we specify the name like other specs using the most appropriate name from the list of error names:  https://heycam.github.io/webidl/#idl-DOMException-error-names?

For example, most other specs say something like "reject promise with a DOMException whose name is "SecurityError".

Reply:
I have added the error names. Not completely sure about these names though :)

Reply:
We'll need to go into more error handling details for the actual standard spec, but this looks good for the explainer.